client/src/components/App.js

```diff
@@ -118,8 +119,13 @@ var App = React.createClass({
       AppActions.setLocation("Connect");
     } else {
       this.setState({ networkName: network.name });
+      const channels = JSON.parse(localStorage.getItem( "anonet.app." + network.user.username + "." + network.name + ".channels")) || [{ 'name': 'ipfs'}];
+      channels.map( (c) => NetworkActions.joinChannel(c.name, ''));
```

Comment:
Use `.forEach` here as you're looping through the channels but not returning anything.